package.json

```diff
@@ -53,7 +53,7 @@
     "lerna": "lerna",
     "test": "jest",
     "test:build:website": "./admin/scripts/test-release.sh",
-    "watch": "yarn lerna run --parallel watch",
+    "watch": "yarn lerna run --parallel watch --concurrency 999",
```

Comment:
unfortunately it doesn't work as I would like and how it used to work.

I want compilation errors to be reported to the console as I change the code live

Now it doesn't report console errors anymore, they are probably swallowed because the process does not end with

<img width="1475" alt="CleanShot 2023-01-26 at 11 36 08@2x" src="https://user-images.githubusercontent.com/749374/214815464-61bb6ce9-158a-43ed-b381-70cee5dbaf7f.png">

Looks like I could use the `--stream` option though:

<img width="860" alt="CleanShot 2023-01-26 at 11 39 00@2x" src="https://user-images.githubusercontent.com/749374/214815872-2913b003-1a73-4154-b43c-2a62ef2bd5af.png">

What I don't like is that it prints memory leaks errors now 😅

<img width="1259" alt="CleanShot 2023-01-26 at 11 39 27@2x" src="https://user-images.githubusercontent.com/749374/214815999-8cfac5b6-6c72-4658-b359-c45f5978ebc8.png">

Reply:
Hey @slorber! Just as a heads up, we just merged the fix for the event emitter warning we were seeing here. After its released I'll get this PR fixed back up.

Reply:
This was fixed after the rebase + bump to latest 🎉 

![Uploading image.png…]()